src/NzbDrone.Core/Download/Clients/Transmission/TransmissionProxy.cs

```diff
@@ -164,15 +160,20 @@ private TransmissionResponse GetTorrentStatus(IEnumerable<string> hashStrings, T
                 "hashString", // Unique torrent ID. Use this instead of the client id?
                 "name",
                 "downloadDir",
-                "status",
                 "totalSize",
                 "leftUntilDone",
                 "isFinished",
                 "eta",
+                "status",
+                "secondsDownloading",
+                "secondsSeeding",
                 "errorString",
                 "uploadedEver",
                 "downloadedEver",
                 "seedRatioLimit",
+                "seedRatioMode",
+                "seedIdleLimit",
+                "seedidleMode",
```

Comment:
This needs to be `"seedIdleMode"` with a capital `I`.

Reply:
Fixed. Tnx for testing, of course.